packages/EasyPipeline/tests/Implementation/Illuminate/IlluminatePipelineTest.php

```diff
@@ -45,7 +45,9 @@ function ($input, $next) {
 
                 return $next($input);
             },
-            [$this, 'actAsMiddleware'],
+            function ($input, \Closure $next) {
+                return $this->actAsMiddleware($input, $next);
+            },
```

Comment:
Non-array methods are better typed. Humans, PHPStorm, PHPStan and Rector underdstand them much better :)